validation/src/test/scala/uk/gov/nationalarchives/tdr/validation/utils/ConfigUtilsSpec.scala

```diff
@@ -0,0 +1,36 @@
+package uk.gov.nationalarchives.tdr.validation.utils
```

Comment:
Could the tests have "should"s to explain the expected outcomes.

Reply:
included shoulds